vda5050_connector/vda5050_connector_py/vda5050_controller.py

```diff
@@ -93,7 +93,8 @@
 DEFAULT_ROBOT_NAME = "robot_1"
 DEFAULT_MANUFACTURER_NAME = "robots"
 DEFAULT_SERIAL_NUMBER = "robot_1"
-DEFAULT_PROTOCOL_VERSION = "2.0.0"
+DEFAULT_PROTOCOL_VERSION = "1.1.0"
```

Comment:
```suggestion
DEFAULT_PROTOCOL_VERSION = "2.0.0"
```

We implemented the connector for v2, based on [this msgs package](https://github.com/ipa320/vda5050_msgs/pull/29). But we might have missed a some fields while reviewing the documentation.